docker-images/kafka/kafka-thirdparty-libs/2.5.x/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I _think_ we should delete the `2.3.x` third-party libs also, if we are removing 2.3.1

Please can someone else confirm this? :)

Reply:
This is correct, according to our conversations with @scholzj as well. Good spot Sam.